docs/devel/flaky-tests.md

```diff
@@ -194,6 +194,41 @@ kubectl delete replicationcontroller flakecontroller
 
 If you do a final check for flakes with `docker ps -a`, ignore tasks that exited -1, since that's what happens when you stop the replication controller.
 
+
+# Reproducing integration test flakes
```

Comment:
It's not only about etcd but also about starting multiple integration test processes. I updated the text a bit.